gax-grpc/src/main/java/com/google/longrunning/stub/OperationsStub.java

```diff
@@ -74,6 +75,10 @@
     throw new UnsupportedOperationException("Not implemented: deleteOperationCallable()");
   }
 
+  public UnaryCallable<WaitOperationRequest, Operation> waitOperationCallable() {
```

Comment:
`@Override`

Reply:
I don't think there's a corresponding method in `BackgroundResource`? Unless you meant that the `@Override` should be added to `GrpcOperationsStub`, in which case, done.

Reply:
Also, I'll be adding this to the GAPIC generation, so this will be consistent with a future generated surface.